walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Pair.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class Pair {
```

Comment:
I would go with just naming this `SchedulingResult`, and just sticking with public member fields for now. I may have also considered just keeping this as a private class in the previous file, since this class is an implementation detail that we won't really share.

Reply:
done